src/providers/timer.provider.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback4-example-log-extension
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Provider} from '@loopback/context';
+import {TimerFn, Time} from '../types';
+
+export class TimerProvider implements Provider<TimerFn> {
+  constructor() {}
+
+  value(): TimerFn {
+    return (start?: [number, number]): Time => {
```

Comment:
It will be much more readable if you define a type such as:
```
export type HighResolutionTime = [number, number]; // [seconds, nanoseconds]
```

Reply:
+1

It's a shame that Node.js typings don't provide such a named type :(